.github/workflows/build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Install Python dependencies
         run: |
           python3 -m pip install --upgrade pip
-          python3 -m pip install torch==1.5.0
+          python3 -m pip install torch==1.5.0 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
@csukuangfj , I changed it to use `torch=1.5.0`, as name `1.5.0+cpu` for MacOS is not on the website of `https://download.pytorch.org/whl/torch_stable.html` (instead it's `1.5.0-cp35/36/37/38` on MacOS). Seems it works well, I'll run the job more times to check. Thanks!